tigerpath/templates/tigerpath/privacy.html

```diff
@@ -35,24 +35,30 @@ <h1 class="mb-3">Privacy Policy</h1>
-          <h6>User Data</h6>
-          <p>We are committed to protecting your privacy. Your data is stored securely, and we will never sell your data to third parties. We only store the data that we need to in order to provide you with the best experience possible in TigerPath. This data includes your nickname, class year, major, and four-year course schedule. By using TigerPath, you are consenting to the storage of this data.</p>
-          <h6>University Data</h6>
-          <p>We collect the course listings data from the OIT Web Feed Registry and the Course Offerings webpage, and we compile the requirements information from the Princeton University Undergraduate Announcement website and the official Princeton University department websites.</p>
-          <h6>Third-Party Services</h6>
-          <p>We are using some third-party services in order to provide you with the best experience possible:<p>
-          <ul>
-            <li>TigerBook API, to automatically populate your name, class year, and major.</li>
-            <li>TigerApps Transcript API, to help you prepopulate your course schedule with courses you've already taken.</li>
-            <li>Google Analytics, to collect aggregated anonymous metadata on how you use the app so we can continually improve it.</li>
-          </ul>
-        </div>
+    <section class="container mb-4 mt-4">
+      <div class="text-section">
+        <p>This privacy policy outlines what data we collect about you (the user), where the course offerings and requirements data in TigerPath comes from, and what other third-party services we are using.</p>
+      </div>
+      <div class="text-section">
+        <h4>Authentication</h4>
+        <p>We protect the entire application underneath CAS authentication, which is the same system you use to sign into other Princeton services, like TigerHub.</p>
+      </div>
+      <div class="text-section">
+        <h4>User Data</h4>
+        <p>We are committed to protecting your privacy. Your data is stored securely, and we will never sell your data to third parties. We only store the data that we need to in order to provide you with the best experience possible in TigerPath. This data includes your nickname, class year, major, and four-year course schedule. By using TigerPath, you are consenting to the storage of this data.</p>
+      </div>
+      <div class="text-section">
+        <h4>University Data</h4>
+        <p>We collect the course listings data from the OIT Web Feed Registry and the Course Offerings webpage, and we compile the requirements information from the Princeton University Undergraduate Announcement website and the official Princeton University department websites.</p>
+      </div>
+      <div class="text-section">
+        <h4>Third-Party Services</h4>
+        <p>We are using some third-party services in order to provide you with the best experience possible:</p>
```

Comment:
I think
_We use some third-party..._
since it's continually, over time, right? It's not temporary.

Reply:
Done